src/view/reviewManager.ts

```diff
@@ -1025,6 +1028,51 @@ export class ReviewManager implements vscode.DecorationProvider {
 		await this.validateState();
 	}
 
+	public async createPullRequest(): Promise<void> {
+		const branchName = this._repository.state.HEAD.name;
+		let potentialTargetRemotes = this._prManager.getGitHubRemotes();
+		let picks: vscode.QuickPickItem[] = potentialTargetRemotes.map(remote => new RemoteQuickPickItem(remote));
+		vscode.window.showQuickPick(picks, {
+			ignoreFocusOut: true,
+			placeHolder: 'Choose a remote which you want to send pull request to'
+		}).then(async (selected: RemoteQuickPickItem) => {
+			if (!selected) {
+				return;
+			}
+			let selectedRemote = selected.remote;
+			const base: any = await this._prManager.getMetadata(selectedRemote.remoteName);
+			let targets = `${base.owner.login}:${base.default_branch}`;
+			vscode.window.showQuickPick([targets]).then(target => {
```

Comment:
```ts
if (!target) {
	return
}
```

Reply:
It doesn't seem like there's a reason to nest `.then`s here, rather than chaining them or using `await`.